docs/user/flyout-menu.rst

```diff
@@ -19,49 +41,34 @@ The flyout menu provides access to the following bits of Read the Docs functiona
 * Links to your :doc:`VCS provider </integrations>` that allow the user to quickly find the exact file that the documentation was rendered from.
```

Comment:
```suggestion
```

The new flyout doesn't show this section.

Reply:
This is still about the original one though.

Reply:
Ah, yeah. I commented this after thinking about deleting old sections/re-structuring the docs to not mention the old stuffs --but I'm not sure if we are removing the documentation for the old flyuot or not yet.